clients/drcachesim/simulator/simulator.h

```diff
@@ -57,6 +58,13 @@ class simulator_t : public analysis_tool_t
     virtual bool process_memref(const memref_t &memref);
 
  protected:
+    void init_knobs(unsigned int num_cores,
```

Comment:
Comment that it indicates whether successful by setting `success`

Reply:
Done.